src/components/Timeout/Timeout.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect, useRef } from 'react';
+import './timeout.css';
+
+export default function Timeout(props) {
+  //track idle time in background
+  //set time in minutes for idle time before modal should appear, and countdown time before auto logout
+  const idleTime = 1;
```

Comment:
milliseconds would be preferable over minutes in development even though it's annoying to read as a human. 

reason being if in the future, we need to do any comparison with Date.now() or a timestamp coming from the API, then it's immediately compatible.

even if we have no plans to do that here ever, we would want our "time variables" to be consistent throughout the app.

Reply:
Okay ms sounds right